packages/flutter_tools/lib/src/commands/analyze.dart

```diff
@@ -331,6 +362,7 @@ linter:
     RegExp allowedIdentifiersPattern = new RegExp(r'_?([A-Z]|_+)\b');
     RegExp constructorTearOffsPattern = new RegExp('.+#.+// analyzer doesn\'t like constructor tear-offs');
     RegExp ignorePattern = new RegExp(r'// analyzer says "([^"]+)"');
+    RegExp conflictingNamesPattern = new RegExp('^The imported libraries \'([^\']+)\' and \'([^\']+)\' cannot have the same name \'([^\']+)\'\$');
```

Comment:
Maybe better would be to exclude the error via options.  Adding `import_duplicated_library_named` should do the trick.  That is, in `optionsBody`:

```
analyzer:
  errors:
    import_duplicated_library_named: ignore
    ...

```

PS: the error code name looks odd to me but it is what it is (defined in `error.dart`).